db/db_impl/db_impl_write.cc

```diff
@@ -1473,6 +1481,27 @@ void DBImpl::MaybeFlushStatsCF(autovector<ColumnFamilyData*>* cfds) {
   }
 }
 
+Status DBImpl::TrimMemtableHistory() {
+  autovector<ColumnFamilyData*> cfds;
+  ColumnFamilyData* tmp_cfd;
+  while ((tmp_cfd = trim_history_scheduler_.TakeNextColumnFamily()) !=
+         nullptr) {
+    cfds.push_back(tmp_cfd);
+  }
+  for (auto& cfd : cfds) {
+    autovector<MemTable*> to_delete;
+    cfd->imm()->TrimHistory(&to_delete, cfd->mem()->ApproximateMemoryUsage());
+    for (auto m : to_delete) {
+      delete m;
+    }
+    if (cfd->Unref()) {
+      delete cfd;
+      cfd = nullptr;
+    }
```

Comment:
I believe we need to install a new super version here. Otherwise, the stale super version will hold the memtables from actually being dropped.

Reply:
This has been resolved, hasn't it?

Reply:
Yes